src/models/converters/transactions/mod.rs

```diff
@@ -99,12 +60,35 @@ impl MultisigTransaction {
                     TokenType::Erc721 => TransactionInfo::Transfer(
                         self.to_erc721_transfer(&token, info_provider).await,
                     ),
-                    _ => TransactionInfo::Custom(self.to_custom(info_provider).await),
+                    _ => TransactionInfo::Custom(
+                        self.custom_decode(info_provider, is_cancellation).await,
+                    ),
                 },
-                _ => TransactionInfo::Custom(self.to_custom(info_provider).await),
+                _ => TransactionInfo::Custom(
+                    self.custom_decode(info_provider, is_cancellation).await,
+                ),
             }
         } else {
-            TransactionInfo::Custom(self.to_custom(info_provider).await)
+            TransactionInfo::Custom(self.custom_decode(info_provider, is_cancellation).await)
+        }
+    }
+
+    async fn custom_decode(
```

Comment:
Just being a bit nit picky, but could we rename this to `to_custom` to maintain the style of the other methods?